src/NavBar.php

```diff
@@ -278,9 +279,6 @@ private function initItemsOptions(): void
         if (isset($this->itemsOptions['class'])) {
             $itemsClass = $this->itemsOptions['class'];
             unset($this->itemsOptions['class']);
-            if (is_array($itemsClass)) {
```

Comment:
Is it not needed?

Reply:
I think it is not necessary, I have thought and not found a case where the options class is an array.

Reply:
Well, I can specify class like `['nav', 'dark']`:

```php
$class = ['nav'];
if ($isDarkMode) {
   $class[] = 'dark';
}
echo Html::blaBla['class' => $class];
```

Reply:
Done revert and add tests.